README.md

```diff
@@ -1,6 +1,8 @@
 # superchain-registry
 
-> ⚠️ This repository is a **work in progress**.  At a later date, it will be proposed to, and must be approved by, Optimism Governance.  Until that time, the configuration described here is subject to change.
+> :warning: This repository is a **work in progress**.  At a later date, it will be proposed to, and must be approved by, Optimism Governance.  Until that time, the configuration described here is subject to change.
+> 
+> :no_entry: We're currently not accepting additional chains to be added to the registry. We'll reopen this process when the repository is ready.
```

Comment:
```suggestion
> :no_entry: We're making some changes to this repository and we've paused adding new chains for now. We'll reopen this process once the repository is ready. The Superchain itself, of course, remains open for business.
```

Reply:
(just so people don't think that we're not adding new chains to the Superchain itself)